src/WebJobs.Script.WebHost/BreakingChangeAnalysis/BlobChangeAnalysisStateProvider.cs

```diff
@@ -19,63 +20,84 @@ internal sealed class BlobChangeAnalysisStateProvider : IChangeAnalysisStateProv
         private readonly ILogger _logger;
         private readonly IConfiguration _configuration;
         private readonly IHostIdProvider _hostIdProvider;
+        private readonly IAzureStorageProvider _azureStorageProvider;
 
-        public BlobChangeAnalysisStateProvider(IConfiguration configuration, IHostIdProvider hostIdProvider, ILogger<BlobChangeAnalysisStateProvider> logger)
+        public BlobChangeAnalysisStateProvider(IConfiguration configuration, IHostIdProvider hostIdProvider, ILogger<BlobChangeAnalysisStateProvider> logger, IAzureStorageProvider azureStorageProvider)
```

Comment:
@fabiocav Any reason for keeping this around, now that V2->V3 is done?

Reply:
We can remove this (as discussed). Since the work is already done, let's keep it so we don't increase the scope.